src/styles/utils/index.ts

```diff
@@ -1644,7 +1644,7 @@ const createStyleUtils = (theme: ThemeColors, styles: ThemeStyles) => ({
                 break;
             case CONST.SEARCH.TABLE_COLUMNS.TAX_AMOUNT:
             case CONST.SEARCH.TABLE_COLUMNS.TOTAL_AMOUNT:
-                columnWidth = {...getWidthStyle(variables.w96), ...styles.alignItemsEnd};
```

Comment:
If we change the width to minWidth, will this expand and fix the problem?

Reply:
We could do something similar, yea. Basically have two pre-defined widths and then check if any expense total has more than a certain number of digits and switch between the two widths dynamically based on that.